manufacturing/electrical/wiring/README.md

```diff
@@ -139,7 +143,16 @@ To connect the solenoid to the PCB, you will need to crimp a 2-pin XH connector
 
 ### Oxygen sensor
 
-**#TODO: what is the right plug for this? @inceptionev**
+| Cable Image | Cable Drawing (PDF)  |
+|:--------:|:--------:|
+| ![Oxygen Sensor Cable](images/OxygenSensorCable.jpg) | [![Oxygen Sensor Cable Drawing](images/OxygenSensorCableDrawingImage.JPG)](drawings/20200209-RespiraWorks-Ventilator-Wiring-OxygenSensorCable.pdf) |
```

Comment:
On second thought, might be good to have the KiCAD source file for that drawing in there as well?

Reply:
This is stuff is still in Altium, though I can include the source file

Reply:
KiCad begins with the Rev A PCB

Reply:
All of these should be fixed now.  Drawing is still in Altium, KiCad starts with PCB Rev A.  Will leave the drawing in PDF only for now because any changes going forward we make will be in scratch in KiCad format.